TweenControllerDemo/TutorialBuilder.swift

```diff
@@ -107,32 +107,32 @@ struct TutorialBuilder {
     
     //MARK: Tutorial Actions
     
-    private static func describeBottomControlsWithVC(_ vc: TutorialViewController, tweenController: TweenController, scrollView: UIScrollView) {
+    fileprivate static func describeBottomControlsWithVC(_ vc: TutorialViewController, tweenController: TweenController, scrollView: UIScrollView) {
         let viewportSize = vc.containerView.frame.size
         let startingButtonFrame = vc.buttonsContainerView.frame
         let startingPageControlFrame = vc.pageControl.frame
-        tweenController.tween(from: startingButtonFrame, at: -1.0)
+        tweenController.tween(startingButtonFrame, at: -1.0)
             .to(startingButtonFrame, at: 0.0)
-            .then(to: CGRect(x: startingButtonFrame.minX, y: viewportSize.height, width: startingButtonFrame.width, height: startingButtonFrame.height), at: 1.0)
-            .thenHold(until: 4.0)
-            .then(to: startingButtonFrame, at: 5.0)
-            .with(action: vc.buttonsContainerView.twc_slidingFrameActionWithScrollView(scrollView))
+            .then(CGRect(x: startingButtonFrame.minX, y: viewportSize.height, width: startingButtonFrame.width, height: startingButtonFrame.height), at: 1.0)
+            .thenHold(4.0)
+            .then(startingButtonFrame, at: 5.0)
+            .with(vc.buttonsContainerView.twc_slidingFrameActionWithScrollView(scrollView))
```

Comment:
I think this block above best illustrates what I'm talking about. The argument labels help.